src/BuildCheck.UnitTests/EndToEndTests.cs

```diff
@@ -401,6 +401,37 @@ void ReplaceStringInFile(string filePath, string original, string replacement)
         }
     }
 
+    [Fact]
+    public void TFMinNonSdkCheckTest()
+    {
+        string content = """
```

Comment:
I'd say this is an example of a project that _shouldn't_ fire this: it's not controlling an output DLL's TF, but it might be doing something on purpose . . .

Reply:
This is now a case where it does NOT fire